source/draw/RasterImage.ooc

```diff
@@ -78,4 +78,5 @@ RasterImage: abstract class extends Image {
 		}
 		result
 	}
+	save: virtual func (filename: String) -> Int { Debug raise("RasterImage save unimplemented for format!") }
```

Comment:
It should be possible to print the class name, too, but I think this will show in the backtrace anyway, so nevermind.